Sources/WebDriver/Poll.swift

```diff
@@ -5,7 +5,7 @@ import struct Dispatch.DispatchTime
 /// Calls a closure repeatedly with exponential backoff until it reports success or a timeout elapses.
 /// Thrown errors bubble up immediately, returned errors allow retries.
 /// - Returns: The successful value.
-internal func poll<Value>(
+public func poll<Value>(
```

Comment:
I think providing a `poll` function is out of scope for swift-webdriver as a library, even if it needs it internally. I'd be curious to see what your use case is though, in case there are functionality gaps to be filled.